app/controllers/news_logs_controller.rb

```diff
@@ -48,7 +48,7 @@ def get_document
   end
 
   def history
-     @histories = @news_log.history_tracks.order(:version => "DESC").page params[:page]
+     @histories = @news_log.history_tracks.order(:version => "ASC").page params[:page]
```

Comment:
we wanted to have the latest change first.

Reply:
Okay I misunderstood the requirement. I will fix it the next commit.